### PR TITLE
fix(codesystem-versions): Support expanding code system versions

### PR DESCRIPTION
**Pull request by AAAlinaaa**

### CODE REVIEW

File: core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/codesystem/CodeSystemConverter.java [hunk 1]
```diff
@@ -73,10 +74,32 @@ public void expand(List<CodeSystem> results) {
 		expandAvailableUpgrades(results);
 		expandExtensionOfBranchState(results);
 		expandUpgradeOfInfo(results);
+		expandVersions(results);
 		
 		// expand additional fields via pluggable converters
 		converters.expand(context(), expand(), locales(), results);
 	}
+	
+	private void expandVersions(List<CodeSystem> results) {
```
Comment (cmark) [inline core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/codesystem/CodeSystemConverter.java]: We already have this feature implemented in the generic resource API. Would it be possible to reuse that logic here as well?
See `com.b2international.snowowl.core.request.ResourceConverter.expandVersions(List<Resource>)`

**Review by cmark — CHANGES_REQUESTED** [review]

File: core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/ResourceConverter.java [hunk 2]
```diff
@@ -132,22 +133,22 @@ private void expandResourcePathLabels(List<Resource> results) {
 		}
 	}
 
-	private void expandVersions(List<Resource> results) {
-		if (expand().containsKey(TerminologyResource.Expand.VERSIONS)) {
-			Options expandOptions = expand().getOptions(TerminologyResource.Expand.VERSIONS);
+	public static void expandVersions(List<? extends Resource> results, Options options, Function<Options, Integer> limit, List<ExtendedLocale> locales, RepositoryContext context) {
```
Comment (cmark) [inline core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/ResourceConverter.java]:
```suggestion
	public static void expandVersions(List<? extends Resource> results, Options expand, Function<Options, Integer> getLimit, List<ExtendedLocale> locales, RepositoryContext context) {
```

**Review by cmark — CHANGES_REQUESTED** [review]

**Review by cmark — APPROVED** [review]
🥓

**Review by apeteri — APPROVED** [review]
👍

**Pull request merged**